article/Method/alpha-k-layer-combination.tex

```diff
@@ -31,3 +31,30 @@ \subsubsection{Optimizing $\alpha_k$}
     \caption{Example of the changed effect to $\alpha_k$. }
     \label{tab:alpha-example}
 \end{table}
+
+\paragraph{Layer effect based on performance} \label{fredsplit}
+After the experiments done on node degree \autoref{subsec:degree-experiment} it was decided to make an algorithm that chose the layers' effect based on how they perform compared to each other.
+It was clearly seen that some layers performed substantially better than others and therefore the worst layers might have a negative effect on the final embedding.
```

Comment:
```suggestion
It could clearly be seen that some layers performed substantially better than others and therefore the worst layers might have a negative effect on the final embedding.
```